storagedriver/testsuites/testsuites.go

```diff
@@ -735,6 +735,27 @@ func (suite *DriverSuite) TestStatCall(c *check.C) {
 	}
 }
 
+// TestPutContentMultipleTimes checks that if storage driver can overwrite the content
+// in the subsequent puts. Validates that PutContent does not have to work
+// with an offset like WriteStream does and overwrites the file entirely
+// rather than writing the data to the [0,len(data)) of the file.
+func (suite *DriverSuite) TestPutContentMultipleTimes(c *check.C) {
+	filename := randomPath(32)
+	contents := randomContents(4096)
+
+	defer suite.StorageDriver.Delete(firstPart(filename))
+	err := suite.StorageDriver.PutContent(filename, contents)
+	c.Assert(err, check.IsNil)
+
+	contents = randomContents(2048) // upload a different, smaller file
+	err = suite.StorageDriver.PutContent(filename, contents)
+	c.Assert(err, check.IsNil)
+
+	readContents, err := suite.StorageDriver.GetContent(filename)
+	c.Assert(err, check.IsNil)
+	c.Assert(readContents, check.DeepEquals, contents)
```

Comment:
This would only really validate the correctness of the `Stat` call, as the contents are checked to be the appropriate length. I'm indifferent about the addition of that.